src/style.css

```diff
@@ -1 +1,102 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf) format('truetype');
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf) format('truetype');
+}
+
+body {
+  font-family: Avenir-Book, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. Consider adding a more generic family like `serif` as a fallback for Avenir-Book.